etc/reset.py

```diff
@@ -172,9 +176,37 @@ def get_pachyderm(deploy_version):
     run("docker", "pull", "pachyderm/pachd:{}".format(deploy_version))
     run("docker", "pull", "pachyderm/worker:{}".format(deploy_version))
 
+def rewrite_config():
+    log.info("Rewriting config")
+
+    keys = set([])
+
+    try:
+        with open(os.path.expanduser("~/.pachyderm/config.json"), "r") as f:
+            j = json.load(f)
+    except:
+        return
```

Comment:
If open does fail, how would you know?

Reply:
Failure here or below most likely means the config does not exist, which means no rewriting is necessary. In the case where there is a failure, it's not a big deal since this is just simplifying the config, and the step can be skipped without negative effect. In the rare case where there is some sort of issue (e.g. the config file is invalid), an error will occur on the next invocation of `pachctl`.

Reply:
Failures can be due to insufficient permissions. Do you want to add logs so it is clear why it failed?

Reply:
In that case, the next invocation of `pachctl` will fail as well